exercises/collatz-conjecture/src/example/kotlin/CollatzCalculator.kt

```diff
@@ -0,0 +1,12 @@
+object CollatzCalculator {
```

Comment:
Not sure what the prevailing philosophy is here (within the track, or within the Kotlin community as a whole) for picking top-level function vs object-scoped function vs class-scoped function vs...!? Direction appreciated :)